packages/flutter/lib/src/widgets/title.dart

```diff
@@ -13,10 +13,11 @@ class Title extends StatelessWidget {
   /// Creates a widget that describes this app to the operating system.
   Title({
     Key key,
-    this.title,
-    this.color,
+    this.title: '',
+    @required this.color,
```

Comment:
changing the API means that the documentation needs to be changed as well, either to correct it or to add the information that should have been there in the first place. :-)

specifically: constructors should warn about any arguments that must not be null, property docs should talk about default values.

Reply:
OK, done.  It feels a bit like documenting what the code already says, but I guess for public docs it's useful to reiterate.